docs/tdvt.md

```diff
@@ -544,3 +544,41 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+## Running TDVT test with tabquerytoolsrv.exe
+You can run a particular TDVT test with the help of a test setup file using tabquerytoolsrv.exe. `tabquerytoolsrv.exe` is located in `C:\Program Files\Tableau\Tableau Server\packages\bin.<version_code>\tabquerytoolsrv.exe` for Windows and `/opt/tableau/tableau_server/packages/bin.<version_code>/tabquerytoolsrv` for Linux. 
+ Before being able to run TDVT tests using tabquerytoolsrv make sure that: 
+   - Your instance of the tableau server is running 
+   - Your server license has been activated 
```

Comment:
Can you link to the tsm command documentation?

Reply:
Adding the link to starting and activating the license.

Reply:
Updated the link